src/js/components/molecules/list-unordered/template.vue.html

```diff
@@ -1,6 +1,7 @@
 <ul class="list">
     <template v-for="(item, index) in listItems.items">
         <template v-if="!item.item"><component :is="listItems.type" :item="item" :key="index"/></template>
```

Comment:
This does not effect the first initial server side render? For the crawler magic to happen. Vue could also get a mismatch in html content and fail on rendering.

Reply:
resolved in discussion with @zwaardje